### PR TITLE
Edit docs/learn/resource-materials/fact-base.md

### PR DESCRIPTION
**Pull request by Phyloiota**

### CODE REVIEW

File: docs/learn/resource-materials/fact-base.md [hunk 1]
```diff
@@ -1,34 +1,15 @@
...
 
+# Fact Base
 
-# Heading 1 (Page Title)
+We are looking for contributors that would like to put together a fact base for IOTA.
 
-content - introduction to the topic
+If You feel you can participate in this topic, please use the editor button below and start contributing.
 
+### Thanks a lot[#](https://wiki.iota.org/docs/learn/networks/network-token-migration#thanks-a-lot)
```
Comment (Critical94) [inline docs/learn/resource-materials/fact-base.md:13]: What is the idea of this link? Can we remove it?
Screenshot_3

Comment (Phyloiota) [reply docs/learn/resource-materials/fact-base.md:15]: ah yes, they are useless, ill remove it

File: docs/learn/resource-materials/fact-base.md [hunk 2]
```diff
@@ -1,34 +1,17 @@
...
-
-<br/>
-
-**need to add a footer section for further content - (three cards)**
-
-## Useful Links
-- **[a link](https://linkgoes.here)** - A description for the link
-- **[a link](https://linkgoes.here)** - A description for the link
-- **[a link](https://linkgoes.here)** - A description for the link
+### Thanks a lot[#](https://wiki.iota.org/docs/learn/networks/network-token-migration#thanks-a-lot)
```
Comment (Phyloiota) [inline docs/learn/resource-materials/fact-base.md:15]:
```suggestion
### Thanks a lot
```

File: docs/learn/resource-materials/fact-base.md [hunk 3]
```diff
@@ -1,34 +1,17 @@
...
 
+### The WIKI Team[#](https://wiki.iota.org/docs/learn/networks/network-token-migration#the-wiki-team)
```
Comment (Phyloiota) [inline docs/learn/resource-materials/fact-base.md:17]:
```suggestion
### The WIKI Team
```

File: docs/learn/resource-materials/fact-base.md (hunk 2 above)
Comment (Dr-Electron) [reply docs/learn/resource-materials/fact-base.md:15]: You need to commit it, not add a note :trollface:

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**